plugin.php

```diff
@@ -130,7 +130,7 @@
 add_filter( 'init', '_add_extra_api_post_type_arguments', 11 );
 add_action( 'init', '_add_extra_api_taxonomy_arguments', 11 );
 add_action( 'rest_api_init', 'rest_register_settings', 0 );
```

Comment:
@websupporter the Settings controller is initialized on `10`, so I set this to `0` to make sure settings were registered before the settings controller was spun up. However, thinking now, I don't think this was necessary, as the settings don't need to be present for the _initialization_ of the settings controller, so I think this could change to the default without issue.